core/src/test/python/compare_model_test.py

```diff
@@ -122,6 +122,54 @@ def testCompareModelFull(self):
 
         self.assertEqual(return_code, 0)
 
+    def testCompareModelSecurityConfiguration(self):
+        _method_name = 'testCompareModelSecurityConfiguration'
+
+        _new_model_file = self._resources_dir + '/compare_model_modela.yaml'
```

Comment:
I do not see these yaml files in the source.  Were they meant to be checked in?

Reply:
added. thanks